docs/java/how-to-register-namespace-in-java.md

```diff
@@ -0,0 +1,38 @@
+---
+id: how-to-register-namespace-in-java
+title: How to register a Namespace in Java
+sidebar_label: Register Namespaces
+description: Use the `RegisterNamespaceRequest` API to register a Namespace and set the Retention Period for the Workflow Execution Event History for the Namespace.
+tags:
+  - developer-guide
+  - java
+  - workers
+---
+
+Use the [`RegisterNamespace` API](https://github.com/temporalio/api/blob/master/temporal/api/workflowservice/v1/service.proto) to register a [Namespace](/concepts/what-is-a-namespace) and set the [Retention Period](/concepts/what-is-a-retention-period) for the Workflow Execution Event History for the Namespace.
+
+```java
+//...
+import io.temporal.api.workflowservice.v1.RegisterNamespaceRequest;
+//...
+public static void createNamespace(String name) {
+    RegisterNamespaceRequest req = RegisterNamespaceRequest.newBuilder()
+            .setNamespace("your-custom-namespace")
+            .setWorkflowExecutionRetentionPeriod(Durations.fromDays(3)) // keeps the Workflow Execution
```

Comment:
Would be nice to also show import to com.google.protobuf.Duration so users know its not java.time